flavors/generic/generic_flavor.go

```diff
@@ -19,13 +19,16 @@ import (
 	"github.com/ligato/cn-infra/httpmux"
 	"github.com/ligato/cn-infra/logging/logrus"
 	"github.com/ligato/cn-infra/servicelabel"
-
 	"github.com/ligato/cn-infra/logging/logmanager"
 	"github.com/ligato/cn-infra/statuscheck"
+	"github.com/ligato/cn-infra/datasync/adapters"
+	"github.com/ligato/cn-infra/datasync/grpcsync"
 )
 
 // FlavorGeneric glues together multiple plugins that are useful for almost every micro-service
 type FlavorGeneric struct {
+	Transports   adapters.TransportAggregator
```

Comment:
No need for aggregator here. We have prepared Aggregator for vpp-agent statistics